functions/mqhandlers.go

```diff
@@ -417,7 +418,20 @@ func updateHostConfig(host *models.Host) (resetInterface, restart bool) {
 	}
 	// store password before updating
 	host.HostPass = hostCfg.HostPass
+
+	// do not update unchangable fields
+	host.DaemonInstalled = hostCfg.DaemonInstalled
+	host.OS = hostCfg.OS
+	host.IPForwarding = hostCfg.IPForwarding
+	host.MacAddress = hostCfg.MacAddress
+	host.Debug = hostCfg.Debug
+	host.PublicKey = hostCfg.PublicKey
+	host.InternetGateway = hostCfg.InternetGateway
+	host.TrafficKeyPublic = hostCfg.TrafficKeyPublic
+	host.Version = hostCfg.Version
+	host.FirewallInUse = hostCfg.FirewallInUse
```

Comment:
I believe these changes are from a different branch?
how are these changes related to this relay PR?